src/test/java/no/sikt/sws/IndexHandlerTest.java

```diff
@@ -177,12 +182,26 @@ void shouldHandleSearchRequestWithQueryParameters() throws IOException {
                 .stripper(WORKSPACEPREFIX)
                 .toJsonCompact();
 
-        assertEquals(compact(testcase.getResponseStripped()), compact(resultBody));
+        assertEquals(compact(testCase.getResponseStripped()), compact(resultBody));
 
         assertThat(response.getStatusCode(), is(equalTo(HTTP_OK)));
     }
 
     private String compact(String body) {
         return body.replaceAll("[\n\r ]", EMPTY_STRING);
     }
+
+    static Stream<TestCaseSws> allRequestArguments() {
```

Comment:
This name should better reflect what the class actually returns/does.

It indicates that it returns all request arguments. But in reality it returns all Search Test Cases